tests/test_cli.py

```diff
@@ -0,0 +1,170 @@
+import os
+import subprocess
+import tempfile
+import pathlib
+import filecmp
+import gc
+
+import pytest
+
+import demes
+from demes.__main__ import cli
+
+CONTINUOUS_INTEGRATION = os.getenv("CI") is not None
+
+
+class TestTopLevel:
+    def test_entrypoint1(self):
+        subprocess.run(
+            "python -m demes -h".split(), check=True, stdout=subprocess.DEVNULL
+        )
+
+    # If demes is "installed", then the "demes" command should be available.
```

Comment:
Not sure about this bit - you're assuming that the module is always installed before we run tests in CI?

Could just test that the command ``demes`` works right after you've pip installed it in CI, and skip this.

Reply:
Done. Thanks!